src/Context/HttpHeadersContext.php

```diff
@@ -95,4 +96,19 @@ public function theHttpHeaderDoesNotContainThePrefixedWord($header, $words) {
     }
   }
 
+  /**
+   * @Then I can see there is a cache HIT at least in one of X-Cache, X-Drupal-Cache, X-Varnish-Cache
+   */
+  public function theCacheHitExists() {
+    try{
+      $this->assertSession()->responseHeaderContains("X-Cache", "HIT");
+    }catch (ExpectationException $exception){
+      try {
+        $this->assertSession()->responseHeaderContains("X-Drupal-Cache", "HIT");
+      }catch (ExpectationException $exception1){
+        $this->assertSession()->responseHeaderContains("X-Varnish-Cache", "HIT");
```

Comment:
the code with nested exception is hard to read. Cannot you simple check the header yourself iwth regular ifs and throw the expectation exceptoin when none of those is there?